src/check-dapi-name.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from 'ethers';
+import { readOperationsRepository } from './utils/read-operations';
+import { runAndHandleErrors } from './utils/cli';
+import { DapiServerContract } from './utils/evm';
+import { loadCredentials } from './utils/filesystem';
+
+const main = async () => {
+  const credentials = loadCredentials();
+  const operationsRepository = readOperationsRepository();
+
+  const dapiNamePromises = Object.entries(operationsRepository.dapis).flatMap(([chainName, dAPI]) =>
+    Object.entries(dAPI).map(async ([dAPIName, dataFeedId]) => {
+      const chainId = parseInt(operationsRepository.chains[chainName].id);
+      if (!chainId) throw new Error(`🛑 Unknown chain name: ${chainName}`);
+
+      if (!credentials.networks[chainName].url) throw new Error(`🛑 No public RPC URL for chain ${chainName}`);
+      const provider = new ethers.providers.JsonRpcProvider(credentials.networks[chainName].url);
+
+      const dapiServerAddress = operationsRepository.chains[chainName].contracts.DapiServer;
+      if (!dapiServerAddress) throw new Error(`🛑 No dapiServer contract address for chain ${chainName}`);
+      const dapiServer = DapiServerContract(dapiServerAddress, provider);
```

Comment:
Nit: DapiServerContract is not a class so it should be lowercased or even be called something like `getDapiServerContract` or `getDapiServerContractInstance`
```suggestion
      const dapiServer = dapiServerContract(dapiServerAddress, provider);
```